styles/global.scss

```diff
@@ -10,9 +10,6 @@ body,
 div#__nuxt,
 div#__layout,
 main {
-  //without this, footer image is above main contents
-  // block hover, click events
-  z-index: 100;
```

Comment:
no side effects if we remove this line?

Reply:
@Gavin-Gong  I guess i'm late to the conversation and this already have been merged but

you can override the z-index of the modal:

https://oruga.io/components/Modal.html#style

check out NeoCheckbox for example how oruga scss vars can be overridden